### ISSUE TITLE
event methods suddenly stopped working

### ISSUE THREAD

**Issue by thebilljustin** [anteriovieira/vue-youtube#62]
Hello, i am developing an app using this package. i tested it yesterday and everything was working fine. when i worked on the app again, the player event method suddenly stopped working. other user defined methods is still working properly but the video defined methods just stopped working

**Comment by Gkiokan** [comment]
Nope. This is an issue with chrome. Check your console in the Issues tab.
And also you can check that the events are working in firefox and safari as before.

Chrome has recently an update for strict blocking js scripts.... check this out.
We must just specify the bookie attributes and voila.

![image](https://user-images.githubusercontent.com/7249224/93377645-95f81f80-f85b-11ea-8f86-02eca26a7d4a.png)

**Comment by Gkiokan** [comment]
@thebilljustin but I've found a easy fix for that.
Just use the nocookie parameter true and you are good.

Here is my working Component
```vue
        <youtube :video-id="id" :player-vars="options" :nocookie="true"
                  style="height: 100%; width: 100%; "
                  ref="youtube"
                  @error="error"
                  @playing="playing"
                  @ended="ended"
                  />
```

**Comment by thebilljustin** [comment]
yes it seems the issue came from google itself. their documentation
released a late warning. everything is working fine now. thank you for the
response
